jupyter-extension/src/plugins/mount/mountDrive.ts

```diff
@@ -118,6 +118,16 @@ export class MountDrive implements Contents.IDrive {
       return this._getCachedContent();
     }
 
+    // Reset cache
```

Comment:
Out of curiosity, why was this moved? It might be relevant to add some context to this comment.

Reply:
I'll add some additional context. Previously the cache was only reset once the user navigated to a new directory that needed the cache not when any change in directory happens. This means if the loading of the files for a directory was still happening when a user unmounted they would get an error shown to them since it would fail to load.